utils/RequestUtil.js

```diff
@@ -71,13 +71,10 @@ var RequestUtil = {
   },
 
   json: function (options) {
+    var usingHangingRequest = Util.isFunction(options.hangingRequestCallback);
     if (options) {
```

Comment:
Fixed by default assigning `options` to an empty object. This means lines were unindented